src/tlo/methods/healthsystem.py

```diff
@@ -182,6 +183,11 @@ def __init__(self, module, *args, **kwargs):
         self._received_info_about_bed_days = None
         self.expected_time_requests = {}
         self.facility_info = None
+        # self.set_equipment_essential_to_run_event({''})  # HSI needs this attribute, but it is not defined in the Base
+        #                                                    class to allow verification of its existence as a test for
+        #                                                    each HSI event, showing that equipment setup was thought
+        #                                                    through for the event.
```

Comment:
```suggestion
        # HSI_Event instances needs the essential equipment to run event to be set using
        # the set_equipment_essential_to_run_event method, but it is not called with
        # a default value in the base class to allow explicit checking if the method
        # was called as a test for each HSI_Event, showing that equipment setup was 
        # thought through for the event.
```

I would avoid having commented code. For a long comment documenting the expectations of how subclasses use the base class like this, I would say possibly makes more sense to add to docstring for the class (multiline string after `class ...:` line, starting `Base HSI event class, from which all others inherit.` above.

Reply:
Just realised this is following pattern `# self.EXPECTED_APPT_FOOTPRINT ... ` comment above, so feel free to ignore.